src/Home.js

```diff
@@ -1,22 +1,85 @@
-import React, { useState, useEffect } from 'react';
-import { StyleSheet, Text, View, FlatList, Pressable, Modal, TextInput, TouchableOpacity } from 'react-native';
-import { DataStore } from 'aws-amplify';
-import { Todo } from './models';
+import React, {useState, useEffect} from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  FlatList,
+  Pressable,
+  Modal,
+  TextInput,
+  TouchableOpacity,
+} from 'react-native';
+import {DataStore} from 'aws-amplify';
+import {Todo} from './models';
```

Comment:
Why are we no longer leaving white space between the brackets?

Reply:
Prettier removed it for jsx file extension. I have fixed it.